corehq/apps/smsforms/tests/test_sql_session.py

```diff
@@ -141,6 +149,99 @@ def test_get_open_sms_session_one_result(self):
         session = SQLXFormsSession.get_open_sms_session(domain, contact)
         self.assertEqual(new_session.session_id, session.session_id)
 
+    @patch('corehq.apps.smsforms.models.utcnow')
```

Comment:
Have we ever looked at using something like https://github.com/spulec/freezegun/ ? Making a special utcnow function seems pretty clunky.

Reply:
i haven't looked into using a third party tool to do this, but in general mocking datetimes is not a straightforward thing to do